portmaster/guides/contribute.md

```diff
@@ -1,13 +1,48 @@
 ---
 title: How To Contribute
+toc_max_header: 3
 layout: base
 ---
 
 ###### Thanks For Participating!
 
-Your contributions will have the highest impact when aligned with Safing's current goals and focus. As a result, these are the current scopes where we are super thankful for your participation:
+It is amazing to see you love Portmaster so much that you are checking out this guide. And honestly, as a small team **your contribution means the world to us.** We could not do it without you!
 
-## Use the Portmaster
+## Spread the Word
+
+> With [version 1.0](https://safing.io/blog/2022/10/27/portmaster-reaches-1.0/), Portmaster should be recommended as one of the first steps anybody should take within their privacy journey.
+
+<div class="notification-warning">
+  <i class="fas fa-fire" style="width:4em;font-size:2em;color:white;text-align:center;padding-top:0.2em;padding-right:0.5em;"></i>
+  <p>
+    As new software - too few people know about Portmaster. <br/> This is where your impact will be biggest!
+  </p>
+</div>
+
+
+### Ask Your Favorite Creators For Reviews
+
+Think of your favorite creators and shows. It could be YouTubers, Newsletters, blogs, guides, lists, radio shows, podcasts and so on. Do these creators know about Portmaster already? Would they enjoy sharing a review on their show? Give them a ping and ask them what they think about Portmaster. This could domino effect into hundreds of thousands of people getting better privacy!
+
+### Do You Have An Audience?
+
+Are you a creator or do you give recommendations? No matter what you think of Portmaster - we'd love for you to have a look and tell your audience what you think!
+
+Optionally, give us a ping after to brighten our day.
+
+### Tell Your Friends
+
+We have no idea how you communicate with your friends on- and offline; and we do not want to know. But what we do know is that if you enjoy Portmaster somebody else will likely too. You know best whom to tell and how to tell them. Be it a private signal message or a recommendation on your favorite online forum.
+
+To all of you. Thanks for sharing, it makes all the difference!
+
+## Financial Support
+
+The most direct way to [support Portmaster development]({{ site.pricing_url }}) is through an upgrade. The "Supporter" package gives you optional badges to show off your support, while ["Unlimited"]({{ site.spn_url }}) takes your privacy to the next level.
+
+## While Using Portmaster
+
+> After [version 1.0](https://safing.io/blog/2022/10/27/portmaster-reaches-1.0/), reporting bugs is no longer as important as it was during the "Alpha" stage. It is still appreciated, but we currently have few resources to fix all edge case bugs out there.
```

Comment:
I do not like telling people not to send bug reports, what we do with them is a different thing, but I would rather go with something like this:

If you find something pleas share it with us, we do have limited resources, but being aware of an issue helps us in making Portmaster a better experience for all. Keep in mind though that we are not at a stage yet, where we can focus on edge cases.